src/coffee.js

```diff
@@ -0,0 +1,312 @@
+const coffee = [
+	{
+		name: "Aynu",
+		tastingProfile: ["Rounded", "Fruit Acidity", "Nut Aroma"],
+		processing: ["Washed"],
+		roastProfile: ["Dark Roast, Medium Roast, Light Roast"],
```

Comment:
This should be an array of strings:
```suggestion
		roastProfile: ["dark roast", "medium roast", "light roast"],
```